optuna/structs.py

```diff
@@ -27,6 +32,11 @@ class TrialState(enum.Enum):
     PRUNED = 2
     FAIL = 3
 
+    def __repr__(self):
+        # type() -> str
+
+        return str(self)
```

Comment:
E.g. `<TrialState.COMPLETE: 1>` -> `TrialState.COMPLETE` to allow whttps://github.com/pfnet/optuna/pull/663/files#diff-fa05647961aebeab39358ccfd19c8e9dR134.